docs/template_plugin/tests/functional/op_reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace reference_tests;
+
+using Attrs = op::v6::ExperimentalDetectronROIFeatureExtractor::Attributes;
+using ExperimentalROI = op::v6::ExperimentalDetectronROIFeatureExtractor;
```

Comment:
I propose to remove these usings. Because you use `Attrs` and `ExperimentalROI` only one time.

Reply:
Fixed.